jooby/src/main/java/org/jooby/handlers/AssetHandler.java

```diff
@@ -253,9 +264,9 @@ private void doHandle(final Request req, final Response rsp, final Asset asset)
     }
 
     // cache max-age
-    if (maxAge > 0) {
-      rsp.header("Cache-Control", "max-age=" + maxAge);
-    }
+    maxAgeOpt.ifPresent(d -> {
```

Comment:
> You change maxAge to be an optional now and think it will be always present, which cause to ALWAYS set the Cache-Control header to a max-age=1

The definition of the field is as followed:

```Java
private Optional<Duration> maxAgeOpt = Optional.empty();
```

So there is initially no value, which means that, by default, the field is not set.